packages/flutter/lib/src/material/banner_theme.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:ui';
```

Comment:
Why is this import needed?

Reply:
It's needed for lerpDouble.

I've changed it to `import 'dart:ui' show lerpDouble;`